ext/storage_sources/azure_store/test/test_azure_connection.cpp

```diff
@@ -31,4 +31,26 @@
 
 #include "azure_connection.h"
 
-TEST_CASE("testing class azure_connection", "azure_connection") {}
+TEST_CASE("testing class azure_connection", "azure_connection")
+{
+    SECTION("List blobs under the test container.") {}
+
+    SECTION("Test object_exists.")
+    {
+        azure_connection conn = azure_connection("myblobcontainer1", "object_exist_test_prefix");
+        bool object_exists;
+
+        // Check for a non-existant object in the container.
+        REQUIRE(conn.object_exists("test.txt", object_exists) == 0);
+        REQUIRE(!object_exists);
```

Comment:
can we expand on this because object_exists isn't defined beforehand? Perhaps we can do object_exists == false or true

Reply:
it doesn't matter what I initialise `exists` to, when it's passed into the `object_exists` function it is set to `false` at the beginning of the implementation. i'll set it to false initially for readability.

Reply:
update: when I initialise `exists`, I wont set it to `false` because the `object_exists` function will do that for me. For the test cases however, I've changed the test condition from `!exists` to `exists == false` and `exists` to `exists == true` for clarity

Reply:
sounds good!